src/governance/governance.cpp

```diff
@@ -726,8 +726,7 @@ void CGovernanceManager::VoteGovernanceTriggers(const std::optional<const CGover
                                                 const CActiveMasternodeManager* const mn_activeman)
 {
     // only active masternodes can vote on triggers
-    if (!fMasternodeMode) return;
-    if (Assert(mn_activeman)->GetProTxHash().IsNull()) return;
+    if (mn_activeman == nullptr || mn_activeman->GetProTxHash().IsNull()) return;
```

Comment:
lets drop this check and replace mn_activeman to the reference same as `CreateGovernanceTrigger`

Reply:
Resolved in latest push!